### DISCUSSION TITLE
Theres a way to disable others cheats and only use the skins unblocked cheat???

### DISCUSSION THREAD

**Discussion by pharita123** [FrozenFish259/Goose_Goose_Duck_Hack#337] [General 聊天区]
I dont wanna know where other players are on the map, walk on walls.. or any these cheats
i just wanna all the skins unblocked

**Comment by FrozenFish259** [comment]
Ok, I'll take this into consideration

**Reply by pharita123** [reply]
Thank u for the hard work!!! ♥
